.appveyor.yml

```diff
@@ -1,3 +1,7 @@
+###
+### .appveyor.yml
```

Comment:
This comment is redundant in my option. I'd suggest removing it.

Reply:
Not redundant on a printout.